integration_tests/integration/probationPractitionerReferrals.spec.js

```diff
@@ -169,12 +170,43 @@ describe('Probation practitioner referrals dashboard', () => {
       serviceCategories: [accommodationServiceCategory, socialInclusionServiceCategory],
     })
 
+    const conviction = deliusConvictionFactory.build({
+      convictionId: 123456789,
+      active: true,
+      offences: [
+        {
+          mainOffence: true,
+          detail: {
+            mainCategoryDescription: 'Burglary',
+            subCategoryDescription: 'Theft act, 1968',
+          },
+        },
+        {
+          mainOffence: false,
+          detail: {
+            mainCategoryDescription: 'Common and other types of assault',
+            subCategoryDescription: 'Common assault and battery',
+          },
+        },
+      ],
+      sentence: {
```

Comment:
The only field on the sentence which affects this class's behaviour is the `expectedSentenceEndDate`, so can we use the `deliusSentence` factory to create this object and omit the other fields? Ditto for the other integration test.